py_trees/display.py

```diff
@@ -183,6 +211,33 @@ def generate_lines(root, internal_indent):
     return s
 
 
+def xhtml_tree(
+        root,
+        show_status=False,
+        visited={},
+        previously_visited={},
+        indent=0,
+     ):
+    """
+    Graffiti your ascii tree on an html snippet.
+
+    Args:
+        root (:class:`~py_trees.behaviour.Behaviour`): the root of the tree, or subtree you want to show
+        show_status (:obj:`bool`): always show status and feedback message (i.e. for every element, not just those visited)
+        visited (dict): dictionary of (uuid.UUID) and status (:class:`~py_trees.common.Status`) pairs for behaviours visited on the current tick
+        previously_visited (dict): dictionary of behaviour id/status pairs from the previous tree tick
+        indent (:obj:`int`): the number of characters to indent the tree
+
+    Returns:
+        :obj:`str`: an ascii tree (i.e. as a html snippet)
+
+    .. seealso:: :meth:`py_trees.display.ascii_tree`
+    """
+    lines = ascii_tree(root, show_status, visited, previously_visited, indent, symbols=html_symbols)
+    lines = lines.replace("\n", "<br/>\n")
```

Comment:
Only need the <br/> here. The \n is not really doing anything useful.

Reply:
I know it doesn't usefully effect the xhtml, but it does logically break it up a little so it's easier to read rather than one giant text blob if you happen to be trying to debug the actual xhtml under the hood.

Reply:
True, but then the xhtml output has these stray \n lines everywhere and the user has to manually replace them after calling this API in order to view the data properly in a browser.

[From Example Above] See here:
https://jsfiddle.net/396pnw4x/show

Reply:
I was being silly. Sorry about that. Decided to copy paste __repr__ instead of the output value